src/test/regress/expected/escape_extension_name_0.out

```diff
@@ -0,0 +1,72 @@
+CREATE SCHEMA "extension'test";
+-- use  a schema name with escape character
+SET search_path TO "extension'test";
+-- this output will help us to understand why we have alternative outputs for this test
+-- print true if uuid-ossp is available false otherwise
+SELECT CASE WHEN COUNT(*) > 0
+  THEN 'SELECT true as uuid_ossp_present'
+  ELSE 'SELECT false AS uuid_ossp_present'
+  END AS uuid_present_command
+FROM pg_available_extensions()
+WHERE name = 'uuid-ossp'
+\gset
+:uuid_present_command;
+ uuid_ossp_present 
+-------------------
+ f
+(1 row)
+
+-- test if citus can escape the extension name
+CREATE EXTENSION "uuid-ossp";
+ERROR:  could not open extension control file "/home/onurctirtir/.pgenv/pgsql-12.0/share/extension/uuid-ossp.control": No such file or directory
```

Comment:
Given that your home directory is in here, will this actually work for anyone else? Maybe use a BEGIN .. EXCEPTION block to hide the error message.

Reply:
that exactly makes sense, sorry for this

Reply:
Updated the test and output files like we did in `custom_aggregate_support.sql`